odoo/addons/base/ir/ir_ui_view.py

```diff
@@ -153,6 +153,7 @@ def remove_element(node):
 xpath_utils['hasclass'] = _hasclass
 
 TRANSLATED_ATTRS_RE = re.compile(r"@(%s)\b" % "|".join(TRANSLATED_ATTRS))
+WRONGCLASS = re.compile(r"(@class\s*=|=\s*@class|contains\(@class)")
```

Comment:
ISTM that including `=\s*@class` makes the regex more complex than necessary, as there wasn't a single occurrence in our codebase.
In fact, this might be an interesting way to allow developers to suppress the warning and still get the intended `=` effect.
We don't have any such workaround for `contains()`, though. Maybe we should remove the warning for `contains` entirely, as the chances of errors are a lot lower.